### PR TITLE
Issue #15340: created InputFormatted file for section 3 Source File

### PR DESCRIPTION
**Pull request by Zopsss**
#15340

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule3sourcefile/InputFormattedSourceFileStructure.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+///////////////////////////////////////////////////////////////////////////////////////////////
+package com.google.checkstyle.test.chapter3filestructure.rule3sourcefile;
+
+// violation 2 lines above ''package' should be separated from previous line.'
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule3sourcefile/InputFormattedSourceFileStructure.java:21]: Bug found in formatter, [3 Source file structure](https://google.github.io/styleguide/javaguide.html#s3-source-file-structure) says:

```
A source file consists of, in order:

License or copyright information, if present
Package statement
Import statements
Exactly one top-level class

Exactly one blank line separates each section that is present.
```

So there should be one blank line separating copyright/license information and package statement but formatter didn't included it and our config was complaining about it

**Review by romani — APPROVED** [review]
ok to merge AFTER  issue on formatter is created

**Comment by Zopsss** [comment]
> ok to merge AFTER issue on formatter is created

issue opened: https://github.com/google/google-java-format/issues/1148

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**